_data/navigation.yml

```diff
@@ -23,4 +25,10 @@ ethics:
       - title: "Ethics FAQ"
         url: /ethics/faq/
       - title: "Ethics Review Questions"
-        url: /ethics/review-questions/
\ No newline at end of file
+        url: /ethics/review-questions/
+
+faq:
```

Comment:
Maybe leave this out? Not sure there'll be more FAQs, so the navigation on left generated by this might be redundant (as in the committees page).

Reply:
Done